code/SiteTree/TweetHolder.php

```diff
@@ -11,16 +12,12 @@
  *
  * @author AzT3k
  */
-class TweetHolder extends Page {
+class TweetHolder extends Page
+{
 
     private static $table_name = 'TweetHolder';
-	private static $can_be_root = true;
+    private static $can_be_root = true;
     private static $allowed_children = array(
         Tweet::class
     );
-
-}
-
-class TweetHolder_Controller extends Controller {
-
```

Comment:
two classes not allowed on the same file, empty controllers not needed.
All other changes are just formatting